src/amp-story-embed.js

```diff
@@ -51,20 +53,24 @@ export class AmpStoryEmbed {
     this.element_ = element;
 
     /** @private {!Document} */
-    this.doc_ = doc;
+    this.doc_ = win.document;
 
-    /** @private {!Array<!HTMLAnchorElement>} */
-    this.stories_ = Array.prototype.slice.call(element.querySelectorAll('a'));
+    /** @private {?Array<!HTMLAnchorElement>} */
+    this.stories_ = null;
 
     /** @private {?Element} */
-    this.rootEl_;
+    this.rootEl_ = null;
 
     /** @private {?HTMLIframeElement} */
-    this.iframeEl_;
+    this.iframeEl_ = null;
   }
 
   /** @public */
   build() {
+    this.stories_ = Array.prototype.slice.call(
```

Comment:
Please address this comment from Justin: https://github.com/ampproject/amphtml/pull/25006#discussion_r336623678

Using the helper will simply import the function when building the final script, it will work just fine outside of AMP.

Reply:
Done.